buildkite/bazel-central-registry/bcr_presubmit.py

```diff
@@ -174,9 +174,12 @@ def add_presubmit_jobs(module_name, module_version, task_configs, pipeline_steps
         )
         commands = [bazelci.fetch_bazelcipy_command(), fetch_bcr_presubmit_py_command(), command]
         queue = bazelci.PLATFORMS[platform_name].get("queue", "default")
-        concurrency = max(1, (CI_RESOURCE_PERCENTAGE * CI_MACHINE_NUM[queue]) // 100)
+        if low_priority:
+            concurrency = 5
```

Comment:
nit, 5 might be higher than what you calculate below, so you might want to calculate the expression below and in case of low_priority use the min of that and 5